js/app.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+var productLineUp = [];
+
+
+function Product (name, path) {
+  this.name = name;
+  this.path = path;
+  this.clicks = 0;
+  this.timesShown = 0;
+  productLineUp.push(this);
+}
+
+productLineUp.getElWithId = function(id) {
+  for (var ii = 0; ii < productLineUp.length; ii++) {
+    var obj = productLineUp[ii];
+    if (obj.name === id) {
+      return obj;
+    }
+  }
+  console.log('Could not find id');
+};
+
+function clickCounter (event) {
+  var idName = event.target.getAttribute('id');
+  var object = productLineUp.getElWithId(idName);
+  object.clicks++;
+  console.log(object);
+}
+
+var clickPic = document.getElementById('pic');
+clickPic.addEventListener('click', clickCounter);
+
+
+
+// var currentPics = [];
+function random (array) {
+  var temp = Math.floor(Math.random() * productLineUp.length);
+  while (array.includes(temp)) {
+    temp = Math.floor(Math.random() * productLineUp.length);
+  }
+  return temp;
+}
+
+
+function renderPic () {
+  var indexed = [];
```

Comment:
if you move the indexed to be outside of the renderPic(), you can use it next time you use renderPic